app/uk/gov/hmrc/hmrcemailrenderer/templates/mods/mods_import_declaration.scala.html

```diff
@@ -58,14 +59,23 @@ <h2>Details of the goods</h2>
                     <th width="60%">Number of items</th>
                     <td>@quantity</td>
                 </tr>
-                <tr>
-                    <th width="60%">Country</th>
-                    <td>@country</td>
-                </tr>
+                @for(cntry <- country) {
```

Comment:
i am not sure we need this for imports

Reply:
It in the current production release so it still need to work after this is released